.auto/measure.sh

```
@@ -0,0 +1,296 @@
+# --- Warm startup median (secondary) ---
+times=()
+for _ in 1 2 3; do
+  t0="$(python3 -c 'import time; print(time.time())')"
+  run_to 20 "$NVIM" --headless \
+    +"lua vim.defer_fn(function() vim.cmd('qa!') end, 500)" \
+    >/dev/null 2>&1 || true
+  t1="$(python3 -c 'import time; print(time.time())')"
+  ms="$(python3 -c "print(int(($t1-$t0)*1000))")"
+  times+=("$ms")
+done
+IFS=$'\n' sorted=($(printf '%s\n' "${times[@]}" | sort -n))
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)